### DISCUSSION TITLE
IO Error

### DISCUSSION THREAD

**Discussion by cdinside** [radubirsan/FlashBrowser#32] [General]
Hy, 

When i upload a file in to a page, after i select it i receive de error "IO Error" and the file is not uploaded. 

The file already uploaded is made from another computer. This error did not appear in the beginning but started some weeks ago, can't say exactly when. 

What can i do to make the error disappear? If you need more data tell me where to give you from.

Thank you!

![io_error](https://user-images.githubusercontent.com/128462171/226548749-cd0aba81-60ee-4dce-abb4-d548478c110f.png)

**Comment by radubirsan** [comment]
please provide link to the page where the upload dose not work
